target/linux/ramips/dts/mt7621_arcadyan_wg630223.dts

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "arcadyan,wg630223", "mediatek,mt7621-soc";
+	model = "Arcadyan WG630223";
+
+	aliases {
+		led-boot = &led_status_green;
+		led-failsafe = &led_status_red;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_green;
+		label-mac-device = &gmac0;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+		bootargs-override = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led_status_blue: led-1 {
+			label = "blue:status";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 14 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_status_green: led-2 {
+			label = "green:status";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 15 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_status_red: led-3 {
+			label = "red:status";
+			color = <LED_COLOR_ID_RED>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		key-restart {
+			label = "reset";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+};
+
+&nand {
+	status = "okay";
+	mediatek,nmbm;
```

Comment:
It's better to add `mediatek,bmt-remap-range`. NMBM should exclude ubi partition. refer to [actiontec web7200](https://git.openwrt.org/?p=openwrt/staging/nbd.git;a=blob;f=target/linux/ramips/dts/mt7621_actiontec_web7200.dts)

Reply:
May I ask:
If the second and third firmware partition uses squashfs/jffs2 layout, shoud I include them in the remap range?

Reply:
yes, it should be `mediatek,bmt-remap-range = <0x0000000 0x05c0000>, <0x1980000 0x3800000>;`.  Maybe we need to use u-boot to restore the original firmware.